### DISCUSSION TITLE
Option to hide metadata caption

### DISCUSSION THREAD

**Discussion by cavrelsilvera** [bpatrik/pigallery2#474] [Q&A]
Is there an option to hide the caption details that are generated from the IPTC-IIM?  This would be especially useful when viewing the picture on a smartphone.

**Comment by bpatrik** [comment]
There is no such feature at the moment.

**Comment by cavrelsilvera** [comment]
Thank you.   
  
  
  
  
  
Cavrel Silvera
  
  
  
  
  

  
  
>

**Comment by cavrelsilvera** [comment]
I was wondering if you could add this as a feature request?.  Maybe something you can add in a future release.  Thanks for your consideration.
